solidity/contracts/Coordinator.sol

```diff
@@ -87,30 +88,47 @@ contract Coordinator {
       _data);
   }
 
+  // This is mostly useful as a sanity check in the #getId test, because the
+  // hash value there is illegible by design.
+  function getPackedArguments(
+    uint256 _payment,
+    uint256 _expiration,
+    uint256 _endAt,
+    address[] _oracles,
+    bytes32 _requestDigest
+                              )
+    public pure returns (bytes)
+  {
+    return abi.encodePacked(_payment, _expiration, _endAt, _oracles, _requestDigest);
+  }
+
   function getId(
     uint256 _payment,
     uint256 _expiration,
+    uint256 _endAt,
     address[] _oracles,
     bytes32 _requestDigest
   )
     public pure returns (bytes32)
   {
-    return keccak256(abi.encodePacked(_payment, _expiration, _oracles, _requestDigest));
+    return keccak256(getPackedArguments(_payment, _expiration, _endAt, _oracles, _requestDigest));
```

Comment:
This is fine, just know that there is a gas cost associated with every method call in Solidity. However, the cost here is only 63 gas.

Reply:
Thanks, good point. We probably will want to remove this kind of code at some point, in that case.